README.org

```diff
@@ -3,14 +3,17 @@
    Before you start, **please take note** of these warnings and comments:
    - **N.B.** The setup script will check out many packages and take a while!
    - **N.B.** You can ignore "error: addinfo_cache" lines. 
+   - **N.B.** These setup instructions now include the STXS workflow
 
    Supported releases:
    - 10_5_0
+   - 10_6_1_patch2 required for the STXS stage 1.1 information
 
    get flashgg:
    #+BEGIN_EXAMPLE
-   cmsrel CMSSW_10_5_0
-   cd CMSSW_10_5_0/src
+   export SCRAM_ARCH=slc7_amd64_gcc700
```

Comment:
we found that this is useful because (at least for some people) the default is now gcc820, and if you follow the steps with that then it won't compile, and it's not at all obvious what is going wrong & how to solve it.

Reply:
ok